demand_energy.md

```diff
@@ -228,14 +228,14 @@ GCAM has the capability to model three technologies which consume thermal and/or
 In GCAM, DACCS technologies indirectly compete with (a) emissions abatement; and (b) other carbon removal technologies such as BECCS and afforestation based on its cost and the subsidy paid for CO2 removal (i.e., the carbon emissions price). This competition is created by defining an additional "no-DAC" technology which does not capture carbon and has zero cost. 
 
 
+<img src="gcam-figs/dac_structure.png" width="500" height="250" /><br/>
 
 
-![image-20210701123037688](C:\Users\fuhr472\AppData\Roaming\Typora\typora-user-images\image-20210701123037688.png)
 
 **Schematic of DACCS technology competition in GCAM's CO2 Removal (CDR) Sector**
 
 
-
+{: .fig}
```

Comment:
Ok, sorry to be a pain but one last thing.  This `{: .fig}` line seems to be causing the next line "Equations" to render weird.  I'm not sure what it is for, can you delete it?

(Maybe open the PR on the JGCRI repo directly next instead of a fork time so I can just make these trivial changes myself instead of having to go back and forth).

Reply:
Just took care of this. 

I'll definitely do this on JGCRI's fork next time